app/Http/Controllers/MembersController.php

```diff
@@ -26,7 +32,8 @@ public function index()
 		$current_members = Member::orderBy('voornaam', 'asc')->whereIn('soort', ['normaal', 'aspirant', 'info'])->get();
 		$former_members = Member::orderBy('voornaam', 'asc')->where('soort', 'oud')->get();
 		return view('members.index', compact('current_members', 'former_members'));
-		if (Auth::user()->can("listOldMembers", Member::class)) {
+		// TODO below code is never executed? Is it better?
```

Comment:
Sure remove it all. I put most of the "additional" rights like "listOldMembers" in the views, my reason being, otherwise the controllers wil be realllllly big and you need to add this "does this exist" logic to views. So I put it there directly.